app/services/send_welcome_email_service.rb

```diff
@@ -10,6 +10,7 @@ def initialize(current_user:)
   def call
     return unless FeatureService.enabled?(:send_emails)
     return if current_user.welcome_email_sent_at
+    return unless current_user.lead_school?
```

Comment:
I have a hunch that `current_user.lead_school?` will return `nil` for _all_ users at this point.
The organisation 'choosing' page is shown after the user signs in, so there'll be no `session[:current_organisation]`.

See: https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/lib/user_with_organisation_context.rb#L29